exporter_test.go

```diff
@@ -230,3 +230,8 @@ func deleteAllFiles() error {
 	}
 	return nil
 }
+
+// backward compatability for time.Now().UnixMilli() in go 1.16 and earlier
+func unixMilli(t time.Time) int64 {
+	return t.UnixNano() / int64(time.Millisecond)
+}
```

Comment:
This is the same code in 2 different files, can you extract it to something like `time_util.go` file?